fdbserver/ptxn/MutableTeamPeekCursor.actor.h

```diff
@@ -113,6 +113,8 @@ class MutableTeamPeekCursor : public BaseClass {
 	}
 
 	const StorageServerStorageTeams getStorageTeamIDs() const { return storageTeamIDs; }
+
+	const KeyRef getStorageServerToTeamIDKey() const { return storageServerToTeamIDKey; }
```

Comment:
We don't need the cursor to provide the key. This can be easily constructed from the serverID.

Reply:
yeah